### PR TITLE
Read unr txyz2 test

### LINKED ISSUE #3: unit test for timeSeries.readUnrTxyz2()
I don't have a good sense of what good unit tests look like. As you recommended, it might be a good first task for you to write a unit test for timeSeries.readUnrTxyz2(). It will help you get started working on something in the project and it will help me see what a good unit test for that function looks like.

Please write the test as a stand-alone program named readUnrTxyz2_test.py and put it in:
TSTools/Test/timeSeries/

### PR DESCRIPTION
**Pull request by jaredonline**
Closes #3

### CODE REVIEW

File: python/tests/timeSeries/readUnrTxyz2_test.py [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
 
-import timeSeries as ts
+from timeSeries import timeSeries as ts
 
 def test_readUnrTxyz2():
-    assert True == True
+    series = ts.TimeSeries()
+    series.readUnrTxyz2("./tests/timeSeries/AREQ.IGS08.txyz2")
+
+    assert series.time.size == 7759
```
Comment (shmashing) [inline python/tests/timeSeries/readUnrTxyz2_test.py:10]: pretty minor, but could the `7759` constant be pulled out into a variable for clarity on what it represents?

File: python/src/timeSeries/timeSeries.py [hunk 2]
```diff
@@ -26,8 +26,7 @@
 DXDYDZ = 'dXdYdZ'
 ENU = 'ENU'
 
-########################################################################
-class timeSeries:
+class TimeSeries:
```
Comment (pkmcfarland) [inline python/src/timeSeries/timeSeries.py]: I like this change. I'm pretty sure in python 'best practices' classes should start with a capital letter. Thanks for catching that.

File: python/tests/timeSeries/readUnrTxyz2_test.py [hunk 3]
```diff
@@ -1,7 +1,21 @@
 import sys
 sys.path.append("./src")
 
-import timeSeries as ts
+from timeSeries import timeSeries as ts
```
Comment (pkmcfarland) [inline python/tests/timeSeries/readUnrTxyz2_test.py]:
```suggestion
from timeSeries import TimeSeries
```

File: python/tests/timeSeries/readUnrTxyz2_test.py [hunk 4]
```diff
@@ -1,7 +1,21 @@
...
 
 def test_readUnrTxyz2():
-    assert True == True
+    series = ts.TimeSeries()
```
Comment (twistmcflip) [inline python/tests/timeSeries/readUnrTxyz2_test.py]:
```suggestion
    series = TimeSeries()
```

**Review by pkmcfarland — CHANGES_REQUESTED** [review]
Looks good. Please make the suggested changes before merging.

File: python/tests/timeSeries/readUnrTxyz2_test.py (hunk 3 above)
Comment (pkmcfarland) [reply python/tests/timeSeries/readUnrTxyz2_test.py]:
```suggestion
from timeSeries import timeSeries as ts
```

**Review by pkmcfarland — APPROVED** [review]

File: python/tests/timeSeries/readUnrTxyz2_test.py [hunk 5]
```diff
@@ -1,7 +1,21 @@
 import sys
 sys.path.append("./src")
 
-import timeSeries as ts
+from timeSeries import TimeSeries
```
Comment (shmashing) [inline python/tests/timeSeries/readUnrTxyz2_test.py:4]: @twistmcflip outta curiosity, why this change? as opposed to the original `from timeSeries import timeSeries as ts`?

Comment (pkmcfarland) [reply python/tests/timeSeries/readUnrTxyz2_test.py:4]: Originally timeSeries was just a module. But @jaredonline changed it so that it is a module inside a package of the same name. I didn't realize that when I made the suggestion for the change.

File: python/tests/timeSeries/readUnrTxyz2_test.py (hunk 1 above)
Comment (jaredonline) [reply python/tests/timeSeries/readUnrTxyz2_test.py:10]: Yea prolly, I can make that change
